cas-server-support-duo/src/main/java/com/duosecurity/duoweb/Util.java

```diff
@@ -7,6 +7,15 @@
 import javax.crypto.spec.SecretKeySpec;
 
 public class Util {
+    /**
+     * TODO: 12/10/15   move to DuoWeb
```

Comment:
Also, note that commons-codec should also be removed since Java8 provides native base64 functionality.